aerleon/lib/naming.py

```diff
@@ -216,10 +217,13 @@ def __init__(
         self.current_symbol = None
         self.services = {}
         self.networks = {}
+        self.fqdn = {}
         self.unseen_services = {}
         self.unseen_networks = {}
         self.port_re = re.compile(r'(^\d+-\d+|^\d+)\/\w+$|^[\w\d-]+$', re.IGNORECASE | re.DOTALL)
         self.token_re = re.compile(r'(^[-_A-Z0-9]+$)', re.IGNORECASE)
+        # https://regexr.com/3g5j0
```

Comment:
The test cases on regexer are nice. Time permitting, might be nice to move them into small Python tests.

Reply:
Done